app/assets/javascripts/discourse/app/controllers/user-activity.js

```diff
@@ -28,13 +28,6 @@ export default Controller.extend({
     this.set("application.showFooter", showFooter);
   },
 
-  @discourseComputed("currentUser.draft_count")
```

Comment:
As I was taking example on what was made for drafts, I noticed we actually didn’t need this code because `currentUser.draft_count` is already a computed property updated through `MessageBus` so it will change automatically even in a template.

We also don’t need to check the value of `count` as it’s already baked in the `I18n` lib. (exactly like in Ruby/Rails).

Reply:
I’ve just seen we have a linter that says we should not use other keys than `one` and `other` for English locales but it’s a bit weird, isn’t it? :thinking: The `I18n` lib already does what needs to be done for pluralization, so why we don’t want to leverage that?

Reply:
The [plural rules for English](https://www.unicode.org/cldr/cldr-aux/charts/29/supplemental/language_plural_rules.html#en) allow only `one` and `other`. Yes, the `I18n` implementations in Rails and JS support `zero` as well. The problem is that translation tools like Crowdin can't handle `zero` in English source strings. They are ignored and can't be translated into any other languages. That's why there is a linter rule that prevents the usage of `zero`.

So, the recommendation is to use a different translation key if you don't want to use the `other` translation when `count` is 0.